src/components/binding/binding-panel.js

```diff
@@ -160,10 +160,10 @@ genotet.BindingPanel.prototype.initChrs_ = function() {
 
 /**
  * Adds a select2 for a new track.
+ * @param {number} trackIndex Index of the added track.
  * @private
  */
-genotet.BindingPanel.prototype.addTrack_ = function() {
-  var trackIndex = this.data.tracks.length - 1;
+genotet.BindingPanel.prototype.addTrack_ = function(trackIndex) {
```

Comment:
It is used in line170, 179, 184, 192 and 198 in this function... Why do you think it is not used?